src/main/java/api/apiwrappers/OrderApiWrapper.java

```diff
@@ -14,9 +14,32 @@ public Order postOrder(Order order) throws IOException {
     }
 
     public Order getOrder(Order order) throws IOException {
-        String endpoint = baseUrl + "/v2/store/order/" + order.getId().toString();
+        String endpoint = baseUrl + "/v2/store/order/" + order.getId();
         
         return (Order)get(order, endpoint);
     }
 
+    public Order getOrderInventory(Order order) throws IOException {
+        String endpoint = baseUrl + "/v2/store/inventory";
+
+        return (Order)get(order, endpoint);
+    }
+
+    public void deleteOrder(Order order) throws IOException {
+        String endpoint = baseUrl + "/v2/store/order/" + order.getId();
```

Comment:
You can add baseUrl + "/v2/store/order/" in a global variable, since it's used inside all the methods:
private String orderEndpoint = baseUrl + "/v2/store/order/";

Reply:
I used:
`private String orderEndpoint = baseUrl + "/v2/store"; `
since I have a method related to inventory:     
```
public Order getOrderInventory(Order order) throws IOException {
        return (Order)get(order, orderEndpoint + "/inventory");
 }
```